frontend/src/components/applyForm/data/SF424_Individual_2_0-V2.0_F668/Form DAT-Table 1.csv

```diff
@@ -0,0 +1,76 @@
+FORM TITLE:,,Application for Federal Domestic Assistance- Individual,,,,,,,,,,,,,,,,

```

Comment:
can we delete this hardcoded data at his point since we're grabbing this from the API?